src/derive.rs

```diff
@@ -114,6 +138,10 @@ pub trait IntoApp: Sized {
     fn into_app<'help>() -> App<'help>;
     /// @TODO @release @docs
     fn augment_clap(app: App<'_>) -> App<'_>;
+    /// @TODO @release @docs
+    fn into_update_app<'help>() -> App<'help>;
```

Comment:
Let's find a good name for this function

Reply:
if you have suggestions I'm open.

Reply:
`into_app_for_update`